pylxd/models/storage_pool.py

```diff
@@ -780,7 +780,12 @@ def create(cls, volume, name=None, expires_at=None):
 
         # Extract the snapshot name from the response JSON in case it was not provided
         if not name:
-            name = operation.resources["storage_volume_snapshots"][0].split("/")[-1]
+            name = operation.resources.get("storage_volume_snapshots")
+            if name is not None:
+                name = name[0].split("/")[-1]
```

Comment:
```suggestion
            if "storage_volume_snapshots" in operation.resources:
                name = operation.resources["storage_volume_snapshots"][0].split("/")[-1]
```
I **think** this feels more Pythonic, WDYT?

Reply:
Indeed using `in` is always very Pythonic. Guess I have become a true Go dev.